metsrw/metadata.py

```diff
@@ -277,8 +283,13 @@ def serialize(self):
         if self.label:
             el.attrib['LABEL'] = self.label
         if self.target:
-            el.attrib[utils.lxmlns('xlink') + 'href'] = \
-                utils.urlencode(self.target)
+            try:
+                el.attrib[utils.lxmlns('xlink') + 'href'] = \
+                    utils.urlencode(self.target)
+            except ValueError:
+                raise exceptions.MetsError(
+                    'Value "{}" (for attribute xlink:href) is not a valid'
+                    ' URL.'.format(self.target))
```

Comment:
If I read this correctly, `MetsError` is the base exception for the module. `ParseError` is the exception that might normally be raised when reading a METS file. I think this is fine, but something like `MetsEncodeError` might read clearer for exceptions where we're encoding data. I think I prefer the verb-y-ness so I think it's just a preference thing.

Reply:
👍